inventory_management_system_api/repositories/unit.py

```diff
@@ -86,3 +110,18 @@ def _is_duplicate_unit(self, code: str, session: ClientSession = None) -> bool:
         """
         logger.info("Checking if unit with code '%s' already exists", code)
         return self._units_collection.find_one({"code": code}, session=session) is not None
+
+    def _is_unit_in_catalogue_category(self, unit_id: str, session: ClientSession = None) -> bool:
+        """Checks if any documents in the database have a specific unit id
+
+        :param unit_id: The ID of the unit being looked for
+        :return: Returns True if 1 or more documents have the unit ID, false if none do
+        """
```

Comment:
```suggestion
        """Checks if any catalogue categories in the database have a specific unit ID

        :param unit_id: The ID of the unit being looked for
        :param session: PyMongo ClientSession to use for database operations
        :return: `True` if 1 or more catalogue categories have the unit ID, `False` otherwise
        """
```